analytics/saas_uploader.go

```diff
@@ -96,19 +100,27 @@ func (s *saasUploader) upload(tenant, fileName string) error {
 		return fmt.Errorf("http.NewRequest: %s", err)
 	}
 
-	req.Header.Set("Expect", "100-continue")
-	req.Header.Set("Content-Type", "application/x-gzip")
-	req.Header.Set("x-amz-server-side-encryption", "AES256")
+	// needs a bare client because udca service account is not authorized for GCS
```

Comment:
This comment isn't very clear on what's happening here and why. Especially because the authorization is handled outside of this module.

Reply:
I just realize. The given client is configured with prometheus metrics. Using a default client without doing anything would cancel those efforts. I suppose I should copy the Timeout and Transport?